### DISCUSSION TITLE
using ArtificilalLandscape( ) function to generate an artificial landscape map for a specific country

### DISCUSSION THREAD

**Discussion by Jeankwizera** [RangeShifter/RangeshiftR-tutorials#21] [Q&A] [answered]
Can I generate an artificial landscape for a specific country using ArtificilalLandscape( ) function in RangeshiftR?

**Comment by annekathleenmalchow** [comment, marked as answer]
Hi,
the ArtificialLandscape() function can only create rectangular landscapes. 
You could, however, run the simulations on an rectangular artificial landscape with the extend of your specific country and from the results use only those that are located within your countries boundaries (this is probably most appropriate if your country has terrestrial borders only). Otherwise, you can always use an external tool to generate a landscape and then import it as an ASCII grid in RangeShiftR using ImportedLandscape(). 
All the best,
Anne
